index.html

```diff
@@ -25,7 +25,7 @@
             <img src="./src/assets/world.png" alt="world icon" />
           </div>
           <div class="content-select">
-            <select>
+            <select class="arrow">
               <option default>Select language</option>
               <option>English</option>
```

Comment:
This is the default option, is not?

Reply:
researching I did not find this attribute `default`. So I adjusted to the design and used the attribute `selected`